src/devices/bus/snes/snes_carts.cpp

```diff
@@ -40,7 +41,7 @@ void snes_cart(device_slot_interface &device)
 	device.option_add_internal("lorom_sgb2",    SNS_LOROM_SUPERGB2);    // SuperGB2 base cart - unsupported
 	device.option_add_internal("lorom_st010",   SNS_LOROM_SETA10);
 	device.option_add_internal("lorom_st011",   SNS_LOROM_SETA11);
-	device.option_add_internal("lorom_st018",   SNS_LOROM);             // Cart + ST018 - unsupported
+	device.option_add_internal("lorom_st018",   SNS_LOROM_ST018);       // Cart + ST018 - unsupported
```

Comment:
Comment here is outdated, probably worth stripping entirely (should be obvious by the device name anyway, that's valid for the other entries too).